whois-client/src/main/java/net/ripe/db/whois/query/QueryFlag.java

```diff
@@ -164,7 +164,13 @@ public enum QueryFlag {
     SHOW_VERSION(new Builder("show-version")
             .withSearchKey("<version-number>")
             .describedAs("Returns historical version of the object")
-            .requiresArgument(Integer.class));
+            .requiresArgument(Integer.class)),
+
+    /* -------------------------------------------------------------------------------------------------------------- */
+
+    ROA_VALIDATION(new Builder("roa-validation")
+            .describedAs("Validate route objects against rpki roas"));
+
```

Comment:
you do need a search key after that ? I mean this flag can only be used with withSearchKey

Reply:
yes, you need a route after that

Reply:
Done